Formula/kube-ps1.rb

```diff
@@ -0,0 +1,32 @@
+class KubePs1 < Formula
+  desc "Kubernetes prompt info for bash and zsh"
+  homepage "https://github.com/jonmosco/kube-ps1"
+  url "https://github.com/jonmosco/kube-ps1/archive/0.4.0.tar.gz"
+  sha256 "0133c5b41609ab8b4e0715f6581a8ee006a979028ddd697bab7a28c02f903db5"
+  head "https://github.com/jonmosco/kube-ps1"
+
+  bottle :unneeded
+
+  depends_on "kubernetes-cli" => :recommended
+
+  def install
+    prefix.install "kube-ps1.sh", "README.md"
```

Comment:
readme is installed automatically

Reply:
Ah ok.  Should I correct this, or will it go in regardless?

Reply:
should correct